### PR TITLE
Use custom User-Agent header unless overriden by check config

### PR DESCRIPTION
**Pull request by benethor**

### CODE REVIEW

File: alamo_worker/cli.py [hunk 1]
```diff
@@ -1,11 +1,20 @@
 # -*- coding: utf-8 -*-
 import os
+import pip
+
+
+def get_alamo_version():
+    dists = pip.get_installed_distributions()
```
Comment (deejay1) [inline alamo_worker/cli.py]: maybe
```python
import pkg_resources
pkg_resources.get_distribution('alamo-worker').version
```
would be a lot simpler and better supported?

Comment (deejay1) [inline alamo_worker/cli.py]: Also I'd move it to `alamo/__init__.py` and set it there for import per usual convention

File: alamo_worker/plugins/graphite.py [hunk 2]
```diff
@@ -93,6 +94,11 @@ def execute(self, check):
                         'target': metric,
                         'from': settings.GRAPHITE_TIME_RANGE,
                         'format': 'json'
+                    },
+                    headers={
+                        'User-Agent': os.environ.get(
+                            'ALAMO_WORKER_VERSION', 'alamo-worker'
```
Comment (deejay1) [inline alamo_worker/plugins/graphite.py]: see above

File: alamo_worker/cli.py (hunk 1 above)
Comment (pnuckowski) [inline alamo_worker/cli.py]: -> `alamo_worker.conf.default_settings.py`

File: alamo_worker/__init__.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+import pkg_resources
```
Comment (pnuckowski) [inline alamo_worker/__init__.py]: Why not to keep this in settings?
`alamo_worker.conf.default_settings.py`
and access to it is via `settings. ALAMO_WORKER_VERSION`

File: alamo_worker/plugins/graphite.py [hunk 4]
```diff
@@ -93,6 +94,9 @@ def execute(self, check):
                         'target': metric,
                         'from': settings.GRAPHITE_TIME_RANGE,
                         'format': 'json'
+                    },
+                    headers={
+                        'User-Agent': ALAMO_WORKER_VERSION
```
Comment (pnuckowski) [inline alamo_worker/plugins/graphite.py]: `'User-Agent': ALAMO_WORKER_VERSION` -> `'User-Agent': settings.ALAMO_WORKER_VERSION`

File: alamo_worker/__init__.py (hunk 3 above)
Comment (benethor) [inline alamo_worker/__init__.py]: this isn't really a setting, but ok...

**Pull request merged**